### ISSUE TITLE
RC2: SA1300 and SA1303 raised for enum values starting with a lower case letter

### ISSUE THREAD

**Issue by qmfrederik** [DotNetAnalyzers/StyleCopAnalyzers#1936]
Consider the following code:

```
public enum SpecialFile
{
    iTunesMetadata
}
```

This raises the following warnings:
1. SA1300	Element 'iTunesMetadata' must begin with an uppercase letter
2. SA1303	Const field names must begin with upper-case letter.

Essentially, duplicating the same warning. In my case, this now requires two source code warnings. One ought to be enough.

**Comment by pdelvo** [comment]
We try to reduce the number of diagnostics reported for the same violation. I think SA1300 should be reported here and SA13003 should not. Enum values are constant, but I think SA1300 fits better

**Comment by sharwell** [comment]
This code is only supposed to report SA1300, per #1859. The discussion regarding SA1300 vs. SA1303 occurred in the Gitter chat on Nov. 27.

**Comment by nbarbettini** [comment]
Grabbing this.
